webapp/src/App.js

```diff
@@ -282,7 +282,22 @@ class AppComponent extends Component {
       user: {},
       collapsed: true,
       eventKey: null,
-      currentEvent: null
+      currentEvent: null,
+      styling: {
+        logo: {
```

Comment:
Is this just temporary while you're experimenting? (styles in the component state rather than the CSS)